sympy/logic/boolalg.py

```diff
@@ -1637,6 +1808,84 @@ def POSform(variables, minterms, dontcares=None):
     return And(*[_convert_to_varsPOS(x, variables) for x in essential])
 
 
+def ANF(variables, truthvalues):
+    """
+    The ANF function converts the list of truth values to
+    Algebraic Normal Form (ANF).
+
+    The variables must be given as the first argument.
+
+    Return True, False, logical And function (i.e., the
+    "Zhegalkin monomial") or logical Xor function (i.e.,
+    the "Zhegalkin polynomial").
+
+    Examples
+    ========
+    >>> from sympy.logic.boolalg import ANF
+    >>> ANF(['x'], [1, 0])
+    Xor(x, True)
+    >>> ANF(['x', 'y'], [0, 1, 1, 1])
+    Xor(x, y, And(x, y))
+
+    References
+    ==========
+
+    .. [2] en.wikipedia.org/wiki/Zhegalkin_polynomial
+
+    """
+    variables = [sympify(v) for v in variables]
+
+    cfs = coeffs(truthvalues)
+    terms = []
+
+    for i, t in enumerate(product([0, 1], repeat=len(variables))):
+        if cfs[i] == 1:
+            terms.append(t)
+
+    return Xor(*[_convert_to_varsANF(x, variables) for x in terms])
+
+
+def coeffs(truthvalues):
```

Comment:
The name <code>zhegalkin_coeffs</code> is too long. <code>anf_coeffs</code> is prettier and shows that it's related to <code>ANF</code> function. Both of these functions, in fact, returns the same thing, the boolean expression in algebraic normal form. Just in different representations.